risc0/zkvm/src/host/api/mod.rs

```diff
@@ -356,6 +356,10 @@ pub struct SessionInfo {
 
     /// The [ExitCode] of the session.
     pub exit_code: ExitCode,
+
+    /// The [ReceiptClaim] associated with the executed session. This receipt claim is what will be
+    /// proven if this session is passed to the Prover.
+    pub receipt_claim: ReceiptClaim,
```

Comment:
Note that, because all the fields on this struct as `pub`, this is technically a breaking change. In particular, it would break users code if the use the `SessionInfo { _ }` syntax to make a `SessionInfo` object themselves. I would be somewhat surprised if anyone is actually doing this, although I would also understand if we do not want to accept that.

Reply:
I don't like the idea of a receipts being associated with execution.